webpack.config.js

```diff
@@ -18,6 +18,10 @@ var shared = {
     }),
   ],
 
+  externals: {
+      'react/lib/ExecutionEnvironment': true
```

Comment:
This is required to get Enzyme working with Webpack.
https://github.com/airbnb/enzyme/issues/47